fedimint-testing/src/lib.rs

```diff
@@ -354,11 +354,11 @@ where
 pub async fn prepare_snapshot<F>(
     snapshot_name: &str,
     prepare_fn: F,
+    parent_dir: String,
```

Comment:
OK... so it is this PR.

Why? :D

Reply:
Do you just mean this should be a `PathBuf`, if so I agree and missed that. Comment doesn't make it super clear.

Reply:
Got rid of `parent_dir` in both `prepare_snapshot` and `validate_migrations`